packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1203,6 +1219,9 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             // The last message processed at the time of summary. If there are no messages, nothing has changed from
             // the base summary we loaded from. So, use the message from its metadata blob.
             message: extractSummaryMetadataMessage(this.deltaManager.lastMessage) ?? this.baseSummaryMessage,
+            createContainerRuntimeVersion: this.metadata.createContainerRuntimeVersion,
+            createContainerTimeStamp: this.metadata.createContainerTimeStamp,
+            lastSummaryCount: this.metadata.lastSummaryCount,
```

Comment:
I have not looked deeply here, but will this work better instead?

return {
   ...this.metadata,
   summaryFormatVersion: 1,
  disableIsolatedChannels: this.disableIsolatedChannels || undefined,
  ...